src/strings/PowerFxRuntimeResources.en-US.resx

```diff
@@ -0,0 +1,220 @@
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="ErrDivByZero" xml:space="preserve">
+    <value>Invalid operation: division by zero.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrRuntimeTypeMismatch" xml:space="preserve">
+    <value>Runtime type mismatch.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrArgumentOutOfRange" xml:space="preserve">
+    <value>Argument out of range.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrInvalidCharValue" xml:space="preserve">
+    <value>Invalid numerical value, must be in 1...255 range.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrBadLanguageCode" xml:space="preserve">
+    <value>Language code '{0}' not supported.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrNonFiniteArgument" xml:space="preserve">
+    <value>Arguments to the '{0}' function must be finite.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrNonFiniteResult" xml:space="preserve">
+    <value>The function '{0}' returned a non-finite number.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrInvalidDateTimeParsingError" xml:space="preserve">
+    <value>The Date/Time could not be parsed.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrInvalidDateTimeError" xml:space="preserve">
+    <value>Invalid date/time value.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrInvalidBooleanFormatError" xml:space="preserve">
+    <value>The value could not be interpreted as a Boolean.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrInvalidColorFormatError" xml:space="preserve">
+    <value>The value could not be interpreted as a Color.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrUnreachableCodeError" xml:space="preserve">
+    <value>Unknown error.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrNotYetImplementedError" xml:space="preserve">
+    <value>Not implemented: {0}.</value>
+    <comment>Error Message.</comment>
+  </data>
+  <data name="ErrInvalidChain" xml:space="preserve">
+    <value>Invalid Chain: {0}.</value>
```

Comment:
>Invalid Chain: {0} [](http://example.com/codeflow?start=11&length=18)

This is a very technical message, we should replace it with something more user-friendly.